src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.Methods.cs

```diff
@@ -201,12 +201,17 @@ private void WriteMethodBody(IMethodDefinition method)
 
             WriteOutParameterInitializations(method);
 
-            if (_forCompilationThrowPlatformNotSupported)
+            if (_platformNotSupportedExceptionMessage != null)
             {
                 Write("throw new ");
                 if (_forCompilationIncludeGlobalprefix)
                     Write("global::");
-                Write("System.PlatformNotSupportedException(); ");
+                if(_platformNotSupportedExceptionMessage.Equals("Default"))
+                    Write("System.PlatformNotSupportedException();");
+                else if(_platformNotSupportedExceptionMessage.StartsWith("SR."))
```

Comment:
Another option is to always write it without quotes. I always have to use SR. And why not?